src/flows/pipes/QueryBuilder/BucketSelector.tsx

```diff
@@ -67,7 +38,14 @@ const BucketSelector: FC = () => {
       </BuilderCard>
     )
   }
-
+  const _selectBucket = (bucket?: string) => {
+    if (bucket) {
+      event('Query Builder Bucket Selected', {
+        usedSearch: !!search.length ? 'yus' : 'nah',
```

Comment:
```suggestion
        usedSearch: !!search.length,
``` 
I've got a sense of humor somewhere, but shouldn't this just be fine?

Reply:
gotta be a string, and stop being boring